testdata/excel2xml/excel2xml-testdata.csv

```diff
@@ -24,18 +24,18 @@ region_0,Region,Region of img_obj_6,,http://rdfh.ch/4123/54SYvWF0QUW6a,,res-rest
 ,,,,,,,,,:hasText,text-prop,,Dies ist ein einfacher Text ohne Markup,utf8,prop-default,,Nochmals ein einfacher Text,utf8,prop-restricted,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-,,,,,,,,,:hasRichtext,text-prop,,"This is <em>bold and <strong>strong</strong></em> text! It contains links to all resources:
-                <a class=""salsah-link"" href=""IRI:test_thing_0:IRI"">test_thing_0</a>
-                <a class=""salsah-link"" href=""IRI:test_thing_1:IRI"">test_thing_1</a>
-                <a class=""salsah-link"" href=""IRI:image_thing_0:IRI"">image_thing_0</a>
-                <a class=""salsah-link"" href=""IRI:compound_thing_0:IRI"">compound_thing_0</a>
-                <a class=""salsah-link"" href=""IRI:partof_thing_1:IRI"">partof_thing_1</a>
-                <a class=""salsah-link"" href=""IRI:partof_thing_2:IRI"">partof_thing_2</a>
-                <a class=""salsah-link"" href=""IRI:partof_thing_3:IRI"">partof_thing_3</a>
-                <a class=""salsah-link"" href=""IRI:document_thing_1:IRI"">document_thing_1</a>
-                <a class=""salsah-link"" href=""IRI:text_thing_1:IRI"">text_thing_1</a>
-                <a class=""salsah-link"" href=""IRI:zip_thing_1:IRI"">zip_thing_1</a>
-                <a class=""salsah-link"" href=""IRI:audio_thing_1:IRI"">audio_thing_1</a>
+,,,,,,,,,:hasRichtext,text-prop,,"This is <em>bold and <strong>strong</strong></em> text! It contains links to all resources:
+                <a class=""salsah-link"" href=""IRI:test_thing_0:IRI"">test_thing_0</a>
+                <a class=""salsah-link"" href=""IRI:test_thing_1:IRI"">test_thing_1</a>
+                <a class=""salsah-link"" href=""IRI:image_thing_0:IRI"">image_thing_0</a>
+                <a class=""salsah-link"" href=""IRI:compound_thing_0:IRI"">compound_thing_0</a>
+                <a class=""salsah-link"" href=""IRI:partof_thing_1:IRI"">partof_thing_1</a>
+                <a class=""salsah-link"" href=""IRI:partof_thing_2:IRI"">partof_thing_2</a>
+                <a class=""salsah-link"" href=""IRI:partof_thing_3:IRI"">partof_thing_3</a>
+                <a class=""salsah-link"" href=""IRI:document_thing_1:IRI"">document_thing_1</a>
+                <a class=""salsah-link"" href=""IRI:text_thing_1:IRI"">text_thing_1</a>
+                <a class=""salsah-link"" href=""IRI:zip_thing_1:IRI"">zip_thing_1</a>
+                <a class=""salsah-link"" href=""IRI:audio_thing_1:IRI"">audio_thing_1</a>

```

Comment:
Don't know what the tool made here, but I compared the diff carefully, and there isn't a semantic change.